packages/services/index.js

```diff
@@ -344,6 +367,16 @@ module.exports = async function start(opts = {}) {
     }
   }
 
+  if (opts.bridge) {
+    if (!(await portInUse(5432))) {
+      console.log('Bridge server requires Postgres to be running on port 5432')
+    } else if (await portInUse(5000)) {
+      console.log('Bridge server already started')
```

Comment:
nit: Telegram attestation and Exchange rate APIs may not work without redis  But could add a env variable to override the requirement of redis when running locally.

Reply:
Let me add a warning. For now I was only thinking having a lightweight bridge setup that requires only the DB, and to be used for identity.